src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -134,7 +134,7 @@ public T ExpectMsg<T>(T expected, Func<T, T, bool> comparer, TimeSpan? timeout =
         /// </summary>       
         public Terminated ExpectTerminated(IActorRef target, TimeSpan? timeout = null, string hint = null)
         {
-            var msg = string.Format("Terminated {0}. {1}", target.Path, hint ?? "");
+            var msg = string.Format("Terminated {0}. {1}", target, hint ?? "");
```

Comment:
Adjusted the failure message printed out when an `ExpectTerminated` call fails, so I can see the UID number.